lib/src/Uncertainty/Distribution/ExtremeValueCopula.cxx

```diff
@@ -0,0 +1,344 @@
+//                                               -*- C++ -*-
+/**
+ *  @brief The ExtremeValueCopula distribution
+ *
+ *  Copyright 2005-2018 Airbus-EDF-IMACS-Phimeca
+ *
+ *  This library is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU Lesser General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  This library is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU Lesser General Public License for more details.
+ *
+ *  You should have received a copy of the GNU Lesser General Public License
+ *  along with this library.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+#include <cmath>
+#include "openturns/ExtremeValueCopula.hxx"
+#include "openturns/RandomGenerator.hxx"
+#include "openturns/PersistentObjectFactory.hxx"
+#include "openturns/Exception.hxx"
+#include "openturns/SpecFunc.hxx"
+#include "openturns/SymbolicFunction.hxx"
+#include "openturns/Brent.hxx"
+
+BEGIN_NAMESPACE_OPENTURNS
+
+CLASSNAMEINIT(ExtremeValueCopula)
+
+static const Factory<ExtremeValueCopula> Factory_ExtremeValueCopula;
+
+/* Default constructor */
+ExtremeValueCopula::ExtremeValueCopula()
+  : CopulaImplementation()
+  , pickandFunction_(SymbolicFunction("t", "1.0"))
+{
+  setName( "ExtremeValueCopula" );
+  // We set the dimension of the ExtremeValueCopula distribution
+  setDimension( 2 );
+  computeRange();
+}
+
+/* Parameters constructor */
+ExtremeValueCopula::ExtremeValueCopula(const Function & pickandFunction)
+  : CopulaImplementation()
+  , pickandFunction_(pickandFunction)
+{
+  setName( "ExtremeValueCopula" );
+  // We set the dimension of the ExtremeValueCopula distribution
+  setDimension( 2 );
+  setPickandFunction(pickandFunction);
+  computeRange();
```

Comment:
You must add
```
setParallel(false);
```
here (and in other constructors), there is no Function::setParallel method.

Reply:
right. I have a hard time trying to fix my Python installation in order to compile the documentation, so I skip some tests from time to time, sorry.